CODE-OF-CONDUCT.md

```diff
@@ -5,28 +5,28 @@
-- There are many unproductive habits that should be avoided in communication. We borrow the Recurse Center’s [“social rules”](https://www.recurse.com/manual#sec-environment): no feigning surprise, no well-actually’s, no backseat driving, and no subtle -isms. Read the preceding link for further discussion.
-- Please keep unstructured critique to a minimum. If you have solid ideas you want to experiment with, make a fork and see how it works. All feedback should be constructive in nature. If you need more detailed guidance around giving feedback, consult [Digital Ocean’s Code of Conduct](https://github.com/digitalocean/engineering-code-of-conduct#giving-and-receiving-feedback).
-- We will exclude you from interaction if you insult, demean or harass anyone. That is not welcome behavior. We interpret the term “harassment” as including the definition in the [Citizen Code of Conduct](http://citizencodeofconduct.org/); if you have any lack of clarity about what might be included in that concept, please read their definition. In particular, we don’t tolerate behavior that excludes people in socially marginalized groups.
-- Private harassment is also unacceptable. No matter who you are, if you feel you have been or are being harassed or made uncomfortable by a community member, please contact one of the channel ops or any of the IODIDE core team members immediately. Whether you’re a regular contributor or a newcomer, we care about making this community a safe place for you and we’ve got your back.
-- Likewise any spamming, trolling, flaming, baiting or other attention-stealing behavior is not welcome.
+- Please avoid using usernames that are overtly sexual or otherwise might detract from a friendly, safe and welcoming environment for all.
+- Respect that people have differences of opinion and that every design or implementation choice carries trade-offs. There is seldom a single right answer.
+- We borrow the Recurse Center’s [“social rules”](https://www.recurse.com/manual#sub-sec-social-rules): no feigning surprise, no well-actually’s, no backseat driving, and no subtle -isms.
+- Please keep unstructured critique to a minimum. If you have solid ideas you want to experiment with, make a fork and see how it works. All feedback should be constructive in nature. If you need more detailed guidance around giving feedback, consult [Digital Ocean’s Code of Conduct](https://github.com/digitalocean/engineering-code-of-conduct#giving-and-receiving-feedback)
+- It is unacceptable to insult, demean or harass anyone. We interpret the term “harassment” as defined in the [Citizen Code of Conduct](http://citizencodeofconduct.org/); if you are not sure about what harassment entails, please read their definition. In particular, we don’t tolerate behavior that excludes people in socially marginalized groups.
```

Comment:
citizencodeofconduct.org 404's for me, web archive crawled it last a year ago:
https://web.archive.org/web/2020*/citizencodeofconduct.org

Reply:
https://github.com/rust-lang/www.rust-lang.org/pull/1215 changed it to a GH link and it's probably better than linking to the Archive.org, we could do that same